src/RedisEnterpriseCache/RedisEnterpriseCache.Autorest/README.md

```diff
@@ -32,12 +32,12 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
-commit: aef78a6d0f0bc49b42327621fc670200d7545816
+commit: edf14cc0a577f6b9c4e3ce018cec0c383e64b7b0
 require:
   - $(this-folder)/../../readme.azure.noprofile.md
 # lock the commit
 input-file:
-  - $(repo)/specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2023-03-01-preview/redisenterprise.json
```

Comment:
···  
- where:
      verb: Update
      subject: ^$|Database
      variant: ^Update$|ViaIdentity$
···
The directive above removes the `UpgradeViaIdentity` of `Update-AzRedisEnterpriseCacheDatabaseDbRedisVersion`. 
    remove: true